packages/kbn-test/src/functional_test_runner/functional_test_runner.ts

```diff
@@ -30,11 +30,13 @@ import {
   setupMocha,
   runTests,
   Config,
+  SuiteTracker,
 } from './lib';
 
 export class FunctionalTestRunner {
   public readonly lifecycle = new Lifecycle();
   public readonly failureMetadata = new FailureMetadata(this.lifecycle);
+  public readonly suiteTracker = new SuiteTracker(this.lifecycle);
```

Comment:
This property seems unused, and it seems we're just using the `SuiteTracker` to execute side effects and mutate the `lifecycle` service. Can we instead move this into the `_run()` method as something like `setupSuiteTracking(this.lifecycle, config)`? Or maybe just pass in the config path? That way we aren't building a somewhat pointless object, we can make the dependency on the `config` service more explicit, and we can even avoid modifying the config service.

Reply:
Thanks, I changed this. Getting rid of the config object simplified a bunch of stuff. I think it was basically left over from early iterations when I was trying to figure out the FTR structure